crypto-ffi/src/core_crypto/epoch_observer.rs

```diff
@@ -140,24 +145,31 @@ impl EpochObserver {
     /// This is extracted as its own function instead of being implemented inline within the
     /// `impl EpochObserver for EpochObserver` block mostly to consolidate error-handling.
     async fn epoch_changed(&self, conversation_id: ConversationId, epoch: u64) -> Result<(), JsValue> {
-        let converation_id = Uint8Array::from(conversation_id.as_slice());
+        let conversation_id = Uint8Array::from(conversation_id.as_slice());
 
         let promise = self
             .epoch_changed
-            .call2(&self.this_context, &converation_id.into(), &epoch.into())?
+            .call2(&self.this_context, &conversation_id.into(), &epoch.into())?
             .dyn_into::<Promise>()?;
         // we don't actually care what the result of executing the notification promise is; we'll ignore it if it exists
         JsFuture::from(promise).await?;
         Ok(())
     }
 }
 
+#[cfg(target_family = "wasm")]
+impl fmt::Debug for EpochObserver {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str("EpochObserver")
+    }
+}
+
```

Comment:
Similar as in https://github.com/wireapp/core-crypto/pull/1156/files#r2104802935, we can use `derive_more` crate here to deriver simple impls.